server.js

```
@@ -61,22 +68,35 @@ function onConnect(socket) {
           socket.emit('action', { type: 'LOGIN_FAILURE' });
         }
       })();
-      break;
-      }
-      /**
-      if (action.type === 'server/sd') {
-        console.log('Got hello data!', action.data);
-        // socket.emit('action', {type:'message', data:'good day!'});
-      }
-       */
-      default: {
-        return true;
-      }
     }
-  });
+    if (action.type === 'server/getPlaylist') {
+      (async () => {
+        try {
+          const values = serverHelpers.parseUserPlaylist(action.playlistData);
+
+          const querypart1 = 'INSERT INTO playlists VALUES($1,$2,$3,$4,$5,$6,$7,$8,$9,$10) ON CONFLICT (id, owner) ';
+          const querypart2 = 'DO UPDATE SET name=$3, description=$4, followers=$5, public=$6, collaborative=$7, ';
+          const querypart3 = 'snapshot_id=$8, "songStats"=$9, "audioFeatures"=$10;';
 
-  // Clears the interval that sends newly generated Api's to the client on disconnect
-  socket.on('disconnect', () => {
-    clearInterval(sendNewClientApi);
+          await pool.query(querypart1 + querypart2 + querypart3, values);
+        } catch (err) {
+          console.log(err);
```

don't swallow errors unnecessarily